test/unit/gateways/worldpay_test.rb

```diff
@@ -58,6 +58,144 @@ def test_exemption_in_request
     assert_success response
   end
 
+  def test_risk_data_in_request
```

Comment:
RuboCop is complaining about this size of this method. Consider moving the setup piece to a separate private method (or more than one!) to appease it.